### ISSUE TITLE
training error

### ISSUE THREAD

**Issue by tranquan687** [dragonfly606/MonoCD#13]
i got the error when training, do you know what the real reason is for the infinity loss?
i train with the default setting, bs=8 on 1 gpu, 
![image](https://github.com/user-attachments/assets/782d664e-1d9d-411f-8f1b-4a5a0475ef4e)
Further, I noticed that the ap_40 is quite low at epoch 90, is it normal? 
i use pytorch 1.11.0+cu11.3, and dcvv2_lastest as this repo https://github.com/lucasjinreal/DCNv2_latest
![image](https://github.com/user-attachments/assets/4680ce0f-3033-4e99-b0f4-7c1bd6edb05d)

**Comment by rlczddl** [comment]
same problem,have you fixed it??

**Comment by tranquan687** [comment]
> same problem,have you fixed it??

not yet

**Comment by tranquan687** [comment]
i used the same environment to train the MonoFlex and got this result, it's quite the same as  [here](https://github.com/zhangyp15/MonoFlex/issues/15), so i think the environment is not the cause of poor results.
Car AP@0.70, 0.70, 0.70:
bbox AP:95.2802, 89.2360, 82.0531
bev  AP:29.9245, 22.5373, 19.5373
3d   AP:20.5539, 14.8230, 12.8907
aos  AP:95.03, 88.65, 81.14

**Comment by rlczddl** [comment]
I am training on a custom dataset, and when increasing the batch size, this issue has not occurred so far.
 By the way, I would like to ask for advice. What do the parameters "C.MODEL.HEAD.REGRESSION_OFFSET_STAT, _C.MODEL.HEAD.REGRESSION_OFFSET_STAT_NORMAL and DEPTH_RANGE" mean, and how can they be obtained when training with one's own data?

**Comment by Jerry9824** [comment]
> 我在训练时遇到了错误，您知道 Infinity Loss 的真正原因是什么吗？我在 1 个 GPU 上使用默认设置 bs=8 进行训练，![image](https://github.com/user-attachments/assets/782d664e-1d9d-411f-8f1b-4a5a0475ef4e)此外，我注意到 ap_40 在 epoch 90 时相当低，这正常吗？我使用 PyTorch 1.11.0+cu11.3，dcvv2_lastest作为这个 repo https://github.com/lucasjinreal/DCNv2_latest ![image](https://github.com/user-attachments/assets/4680ce0f-3033-4e99-b0f4-7c1bd6edb05d)

你好，我们完全一样的环境一样的问题，请问你已经修复好了吗？

**Comment by a1037441813** [comment]
我出现了相同的问题，，loss中出现了nan 报错
